src/utils/security.cljs

```diff
@@ -0,0 +1,57 @@
+(ns utils.security
```

Comment:
Why is this copied from `status-im.utils.security` @siddarthkay? What do we actually gain by this duplication? I only see it increasing our tech debts. Namespace duplication is a costly practice for maintenance, and we're doing this all the time (not only in this PR).

As far as I've checked, this PR only needs access to `safe-unmask-data` function, which is perfectly implemented in `status-im.utils.security`.

If a contributor wants to actually extract some code into a new namespace to follow the new guidelines, I think they should try to move it properly, not duplicate it. We have tools to automate much of the work.

I have the comment for the duplicated `utils.security-html` namespace created in this PR. It seems because of the new code structure guidelines we turned an (usually) unacceptable practice into the norm.

Reply:
Hmm yes what you say makes sense, @ilmotta 

I'll go ahead and migrate security related stuff to status-im2 namespace in another small PR first.

This would include modifying every other code that references the old namespace to use the new namespaced reference.

This approach sound good?

Reply:
That would be great @siddarthkay :partying_face: We move a full step ahead and avoid leaving traces of tech debt behind us!

Reply:
https://github.com/status-im/status-mobile/issues/14551

Reply:
Removed `security.cljs` and `security-html.cljs` from this PR, I will add them back as part of a separate PR.